### DISCUSSION TITLE
a bug about submod

### DISCUSSION THREAD

**Discussion by monibun** [Monika-After-Story/MonikaModDev#8002] [QA - code and content creation (technical)]
so I'm trying to my make a submod this my codeing
init 5 python:
    addEvent(Event(persistent.event_database,eventlabel="monika_Jane",category=['You'],prompt="You",random=True))

label monika_Jane:
    m 3eub "[player], do you khow a game called Friday Night Funkin?"
    m 2wuo "It's a very popular rythem game!"
    m 6fta "I don't khow if you play it or not..."
    m 1tfd "Did you khow that they make a mod about me too?"
    m 3suc "You better play that mod RIGHT NOW!!"
    m 1hblb "haha."
    m 1nka "Don't worry you can play my mod anytime you want!."
return

so i put it in the submod section when i open the game and search it up it doesn't show up my submod plz anybody help??

**Comment by multimokia** [comment]
You made it a random topic. As such, Monika has to bring it up before it shows up in repeat conversation.

**Comment by thhtthht** [comment]
> You made it a random topic. As such, Monika has to bring it up before it shows up in repeat conversation

But what happen if Monika don't mention it?

**Comment by multimokia** [comment]
She will

**Comment by thhtthht** [comment]
> She will

but what if she don't mention it in a long time?

**Comment by briaryoung** [comment]
You’ll just have to wait. It’s not immediate.

**Comment by thhtthht** [comment]
> You’ll just have to wait. It’s not immediate.

But what if I wait for 1 day and she dont talk about it?

**Comment by briaryoung** [comment]
Then you’ll wait longer.

**Comment by Booplicate** [comment]
Wait first and if she doesn't bring it up, then report it. What's the point of talking of something hypothetical?

**Comment by thhtthht** [comment]
> Then you’ll wait longer.

K

**Comment by Booplicate** [comment]
> so I'm trying to my make a submod this my codeing init 5 python: addEvent(Event(persistent.event_database,eventlabel="monika_Jane",category=['You'],prompt="You",random=True))
> 
> label monika_Jane: m 3eub "[player], do you khow a game called Friday Night Funkin?" m 2wuo "It's a very popular rythem game!" m 6fta "I don't khow if you play it or not..." m 1tfd "Did you khow that they make a mod about me too?" m 3suc "You better play that mod RIGHT NOW!!" m 1hblb "haha." m 1nka "Don't worry you can play my mod anytime you want!." return
> 
> so i put it in the submod section when i open the game and search it up it doesn't show up my submod plz anybody help??

You have multiple issues with your code. So first fix those issues. You didn't show us the code where you define it. For your submod to appear in the submod menu, you have to define it (look for other submods for examples).
